packages/transformer/src/BigMap.ts

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { Id64String } from "@itwin/core-bentley";
+
+export class BigMap<V> extends Map<Id64String, V> {
```

Comment:
document that this is a stopgap around the 16.7M limit of elements in the v8 implementation of JavaScript Map, and that we intend to replace the map usage with a temporary sqlite database anyway. You should link to #83

Reply:
Added docstring